pkg/storageincentives/redistributionstate_test.go

```diff
@@ -43,6 +43,7 @@ func TestState(t *testing.T) {
 	input := Status{
 		Phase:           commit,
 		IsFrozen:        false,
+		IsFullySynced:   false,
```

Comment:
because the default is false anyways, it would be better to have the input as `true`, also the same for IsFrozen